dwave/cloud/computation.py

```diff
@@ -526,7 +526,35 @@ def occurrences(self):
         to this property is non-blocking.
 
         Returns:
-            list or numpy matrix of doubles: Occurrences.
+            list or NumPy matrix of doubles: Occurrences. When returned results are
+            ordered in a histogram, `occurrences` indicates the number of times a particular
+            solution recurred.
+
+        Examples:
+            This example creates a solver using the local system's default D-Wave Cloud Client
+            configuration file, submits a simple Ising problem with several ground states to a
+            remote D-Wave resource for 20 samples, and prints the returned results, which
+            are ordered as a histogram. The problem's ground states tend to recur frequently,
+            and so those solutions have `occurrences` greater than 1.
+
+            >>> from dwave.cloud import Client
+            >>> with Client.from_config() as client:  # doctest: +SKIP
+            ...     solver = client.get_solver()
+            ...     solver.parameters['answer_mode']='histogram'
```

Comment:
Instead of modifying solver parameters, supplying the value as kwarg to sampling method seems cleaner:
```
computation = solver.sample_ising({}, quad, num_reads=500, answer_mode='histogram')
```
Can we use that instead?